app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Employment.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BcGov.Fams3.SearchApi.Contracts.Person
+{
+    public class Employment : PersonalInfo
+    {
+        public string IncomeAssistance { get; set; }
+
+        public string IncomeAssistanceStatus { get; set; }
+
+        public Employer Employer { get; set; }
+
+        public string Occupation { get; set; }
+
```

Comment:
missing  employmentConfirmed ?

Reply:
I think it's funny that you have employment information and ask if employment is confirmed. That field is redundant to me

Reply:
If it is in ICBC data, we need to transfer it. that's what I think.

Reply:
We don't want to be tightly coupled with ICBC though. The presence of the data is a confirmation of employment except the fields means something else

Reply:
We are not sure all the employments are confirmed.

Reply:
Maybe I don't understand, what does that field mean ? that someone is employed or not ?

Reply:
i think icbc got employment info , for example, from applicant. But this employment info is not confirmed through other ways. So, search officer should not be 100% sure this the correct employment info.

Reply:
That makes sense. Information may not be correct. Would update

However we don't have that information on the UI (current UI). It still seems like if employment information is not real, it won't be available. We should be careful not to be tightly coupled with data providers schema cos we would be creating technical debt.

Reply:
Updated